generators/client/templates/angular/package.json.ejs

```diff
@@ -87,6 +87,8 @@
 <%_ } _%>
 <%_ if (cypressTests) { _%>
     "cypress": "<%= dependabotPackageJson.devDependencies['cypress'] %>",
+    "cypress-audit": "<%= dependabotPackageJson.devDependencies['cypress-audit'] %>",
+    "lighthouse": "<%= dependabotPackageJson.devDependencies['lighthouse'] %>",
```

Comment:
Inject the dependency like we did at cypress-coverage:
https://github.com/jhipster/generator-jhipster/blob/6b532305ee32497e6a0ed30c3f1140b34d520d99/generators/cypress/index.js#L144-L160

No need to change at every client framework package.json, even blueprints.